packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -6659,15 +6656,35 @@ class _CopySelectionAction extends ContextAction<CopySelectionTextIntent> {
 
   @override
   void invoke(CopySelectionTextIntent intent, [BuildContext? context]) {
+    if (!state._value.selection.isValid || state._value.selection.isCollapsed) {
+      return;
+    }
+
+    if (!state.widget.selectionEnabled) {
+      return;
+    }
+
     if (intent.collapseSelection) {
       state.cutSelection(intent.cause);
     } else {
       state.copySelection(intent.cause);
     }
   }
+}
+
+class _PasteSelectionAction extends ContextAction<PasteTextIntent> {
+  _PasteSelectionAction(this.state);
+
+  final EditableTextState state;
 
   @override
-  bool get isActionEnabled => state._value.selection.isValid && !state._value.selection.isCollapsed;
+  void invoke(PasteTextIntent intent, [BuildContext? context]) {
+    if (!state.widget.selectionEnabled) {
+      return;
+    }
```

Comment:
Can you add something to the docs of enableInteractiveSelection that explains the behavior of shortcuts when disabled?

Reply:
I think this is already covered by the existing docs:

https://github.com/flutter/flutter/blob/5b6b62183f72271a4e86fe7374602744a2ed0f61/packages/flutter/lib/src/widgets/editable_text.dart#L1723-L1725

Do you want me to expand on that?

Reply:
Oh, oh, I misunderstood your feedback. Good idea, will update!

Reply:
Updated!